submodules/k8s/templates/k8s-functions.sh.tftpl

```diff
@@ -67,8 +67,12 @@ check_remove_calico() {
   if [ -n "$calico_deploy" ]; then
     managed_by=$(kubectl --kubeconfig "$kubeconfig" get deployment $OPERATOR_DEPLOYMENT_NAME -n $OPERATOR_NAMESPACE -o jsonpath="{.metadata.labels['app\.kubernetes\.io/managed-by']}")
     if [ -z "$managed_by" ] || [ "$managed_by" != "Helm" ]; then
-      echo "Removing existing Calico operator..."
       ## Best Effort Remove
+      echo "Removing Calico CRDs..."
+      kubectl_cmd delete installation default --ignore-not-found=true || true
+      sleep 600
```

Comment:
why are we guessing? it is fine if there is an upper time bound, but we should know what we expect in the end.
if its the next line, then do that in a retry until we timeout or see the expected result.

Reply:
yep.